gridt/src/app/core/Signal.model.ts

```diff
@@ -0,0 +1,8 @@
+export interface Signal {
+
+//Last signal
+    
```

Comment:
Why is there so much whitespace?

Reply:
To not look clustered. Tho I do have the tendency to overuse white space

Reply:
I would say that it is a little too much here.